_source/logzio_collections/_log-sources/zipkin.md

```diff
@@ -35,13 +35,17 @@ For most users, these won't be an issue, but they're still good to know:
 
 1.  Download Zipkin server and Zipkin-Logz.io Trace Storage
 
-    Download [Zipkin server](https://search.maven.org/remote_content?g=io.zipkin.java&a=zipkin-server&v=LATEST&c=exec).
+    Download Zipkin server.
 
     ```shell
-    wget -O zipkin.jar 'https://search.maven.org/remote_content?g=io.zipkin.java&a=zipkin-server&v=LATEST&c=exec'
```

Comment:
Just leaving a note that the latest versions of the Zipkin Server are published under a different group id (`io.zipkin` instead of `io.zipkin.java`), so without this pull request merged, the documentation will lead users to download not the latest version of the Zipkin Server. The quickstart script will download the latest, as expected.
See https://github.com/openzipkin/openzipkin.github.io/issues/136

Reply:
Thanks for the note. Should be good to send this live soon.